pages/common/streamlink.md

```diff
@@ -0,0 +1,36 @@
+# Streamlink
+
+> Extracts streams from various services and pipes them into a video player of choice.
+> More information: <https://streamlink.github.io>.
+
+- Attempt to extract streams from the URL specified, and if it's successful, print out a list of available streams to choose from:
+
+`streamlink {{example.com/stream}}`
+
+- Open a stream with specified quality:
+
+`streamlink {{example.com/stream}} {{720p60}}`
+
+- Use **best** or **worst** for selecting the highest or lowest available quality:
+
+`streamlink {{example.com/stream}} {{best}}`
```

Comment:
Perhaps this will read better?
```suggestion
- Select the highest or lowest available quality:

`streamlink {{example.com/stream}} {{best|worst}}`
```